workflow/rules/mapping.smk

```diff
@@ -214,3 +214,126 @@ rule apply_bqsr:
         java_opts="",  # optional
     wrapper:
         "v2.3.2/bio/gatk/applybqsr"
+
+
+rule map_reads_vg_giraffe:
+    input:
+        reads=get_map_reads_input,
+        idx="resources/pangenome/hprc-v1.0-mc-grch38.xg",
```

Comment:
This should not be hardcoded. What if a different pangenome is used? The pangenome index should be configurable via the config.

Reply:
I added `config["link_to_pangenome_idx"]` to use in the download rule and download output path to common_smk.